packages/jest-runner/package.json

```diff
@@ -39,10 +39,11 @@
     "throat": "^6.0.1"
   },
   "devDependencies": {
+    "@tsd/typescript": "~4.6.2",
     "@types/exit": "^0.1.30",
     "@types/graceful-fs": "^4.1.2",
     "@types/source-map-support": "^0.5.0",
-    "jest-jasmine2": "^28.0.0-alpha.8"
```

Comment:
Huh

Reply:
Seems like it does nothing. Dev dependency, so I though it is maybe used in tests or its types are useful? Nope. It is require.resolved. Alright, but on user side dev dependency will not be installed. Do I miss something?

Reply:
If we `require.resolve` it, it should stay. Otherwise stricter dep handling (like yarn pnp) will fail